### PR TITLE
Update apt sources before install

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

**Comment by TheTripleV** [comment]
Fixes CI latex install.

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
         pip install -r source/requirements.txt
     - name: Install LaTeX
       run: |
+        sudo apt-get update
```
Comment (Daltz333) [inline .github/workflows/CI.yaml:18]: Is it just me or is there an extra space?

Comment (Daltz333) [reply .github/workflows/CI.yaml:18]: just me lol

Comment (TheTripleV) [reply .github/workflows/CI.yaml:18]: It looks fine to me.
Screen Shot 2020-11-28 at 9 32 16 PM

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**